corehq/form_processor/models.py

```diff
@@ -365,7 +365,6 @@ class Meta:
 
 class AbstractAttachment(DisabledDbMixin, models.Model, SaveStateMixin):
     attachment_id = UUIDField(unique=True, db_index=True)
-    name = models.CharField(max_length=255, db_index=True, default=None)
```

Comment:
kind of annoying that you can't keep this in the base class if you want to index differently (not proposing anything, just venting)